cmd/pj-rehearse/server.go

```diff
@@ -309,7 +309,10 @@ func (s *server) handlePotentialCommands(pullRequest *github.PullRequest, commen
 				s.rehearsalConfig.AbortAllRehearsalJobs(org, repo, number, logger)
 			default:
 				if rehearsalsTriggered {
-					// We don't want to trigger rehearsals more than once per comment
+					message := fmt.Sprintf("@%s: requesting more than one rehearsal in one comment is not supported", user)
```

Comment:
I don't know that this is a necessary thing to surface to the user. We are still triggering their rehearsals here, we are just not going to trigger them multiple times redundantly. Did you notice some case where this resulted in a completely ignored command?

Reply:
Yes. I asked for N rehearsals and I did not get them. I had no idea why and started wondering if my configuration in the PR was broken. `pj-rehearse` is commonly not telling the user what's going on, which makes people think it's broken or has flakes - anecdotally I have met many people who just try something many times over because it's so opaque.

Reply:
Maybe if you tried to rehearse single job in multiple different lines it would ignore all, but the first one?

Reply:
Can you link to the specific comment? I would like to see if there is a gap that we missed here.

Prow plugins are complicated for sure, so this is a hard thing to do perfectly, but I would like to improve the communication if possible.

Reply:
There were likely other issues involved, since there were no jobs triggered at all for a while, with no comment as to why - perhaps error cases in handling the comment that don't lead to comment responses. Then, we saw the case where only the first got triggered. 

https://github.com/openshift/release/pull/48608#issuecomment-1942036110